pepper-apis/dss-core/src/test/java/org/broadinstitute/ddp/util/TestDataSetupUtil.java

```diff
@@ -621,7 +622,7 @@ public static UserProfile createTestingProfile(Handle handle, long userId, boole
                 .preferredLangId(LanguageStore
                         .get(TestConstants.TEST_USER_PROFILE_PREFERRED_LANGUAGE)
                         .getId())
-                .preferredLangCode(null)
+                .preferredLangCode(TEST_USER_PROFILE_PREFERRED_LANGUAGE)
```

Comment:
Fixing a bug that has been with us for a while: the generated profile data had a language id, but not a corresponding language code, which was leading to breakages in the test when asserting that the test user's preferred language had been reset properly (in addition to it not reflecting the reality of the dao, that uses the language id to lookup and populate the language code).